src/main/java/games/strategy/triplea/ui/menubar/ViewMenu.java

```diff
@@ -308,15 +302,10 @@ private void addShowMapBlends(final JMenu menuGame) {
       TileImageFactory.setShowMapBlends(showMapBlends.isSelected());
       TileImageFactory.setShowMapBlendMode(uiContext.getMapData().getMapBlendMode());
       TileImageFactory.setShowMapBlendAlpha(uiContext.getMapData().getMapBlendAlpha());
-      final Thread t = new Thread("Triplea : Show map Blends thread") {
-        @Override
-        public void run() {
-          frame.setScale(uiContext.getScale() * 100);
-          yield();
```

Comment:
This call to Thread#yield has been removed, see other removed call for the reason

Reply:
This instance was introduced in 7fbaa2f, and the commit message only says, "Various fixes and Map Blending."